src/main/java/io/reactivex/Flowable.java

```diff
@@ -15814,21 +15798,16 @@ public final Completable switchMapCompletableDelayError(@NonNull Function<? supe
     }
 
     /**
-     * Returns a Flowable that only emits those items emitted by the source Publisher that are not followed
-     * by another emitted item within a specified time window, where the time window is governed by a specified
-     * Scheduler.
+     * Alias to {@link #debounce(long, TimeUnit, Scheduler)}
```

Comment:
Please keep the original first sentence, similarly as above.

Reply:
moved the mention of alias to the end of the sentence